get_data_endpoints/README.md

```diff
@@ -4,11 +4,14 @@ This is a component that can be used in Kubeflow Pipelines to read two input dat
 processing is done on that data **in a secure and governed fashion by leveraging Fybrik**
 
 It is assumed that Fybrik is installed together with the chosen Data Catalog and Data Governance engine, and that:
+
 * training and testing datasets have been registered in the data catalog
 * governance policies have been defined in the Data Governance engine
 
 ## Prerequisits
+
 * [Install kubeflow pipelines](https://www.kubeflow.org/docs/components/pipelines/installation/overview/#kubeflow-pipelines-standalone)
+  * To install on kind, see [these instructions](https://github.com/machine-learning-exchange/mlx/blob/main/docs/install-mlx-on-kind.md#install-kubeflow-pipelines-for-reference-only-optional)
```

Comment:
It takes quite a bit of time until all the pods are running and active.  Sometimes the pods crash and restart because there are dependencies on other pods that aren't ready yet.  Give it time.  :)

Reply:
The problem is not in the running or restarting pods, the CRDs are not installed. 
And the next command `kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io` returns a similar error:
```
Error from server (NotFound): customresourcedefinitions.apiextensions.k8s.io "applications.app.k8s.io" not found
```

Reply:
Something in the instructions is wrong.

Reply:
It works with an old K8s (v1.19),
the `creaet cluster` kind command should be 
kind create cluster --image=kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729   --name kfp

Reply:
For a newer K8s, the PIPELINE_VERSION should be 1.8.5, please mention it as a workaround, before the KFP installation guide will be updated.